lib/house.rb

```diff
@@ -0,0 +1,42 @@
+class House
+
+  attr_reader :price, :address, :rooms
+
+  def initialize(price, address)
+    @price = price[1..-1].to_i
+    @address = address
+    @rooms = []
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def above_market_average?
+    if @price > 500000
+      true
+    else
+      false
+    end
+  end
+
+  def area
+    room_areas = []
+    @rooms.each do |room|
+      room_areas << room.area
+    end
+    room_areas.sum
```

Comment:
This is great, and works well.  `Sum` is an enumerable of its own, and could have been used instead of `each. 

```
 @rooms.sum do |room|
      room.area
  end
```

it would iterate over each room and sum the area of each room to get you the total

Reply:
Woahhh! I didn't know sum was also an enumerable! Thanks for calling that out!